iotc-azrtos-sdk/src/iotconnect.c

```diff
@@ -225,19 +226,19 @@ static void on_message_intercept(IotclEventData data, IotclEventType type) {
         sync_response = NULL;
         discovery_response = run_http_discovery(config.cpid, config.env);
         if (NULL == discovery_response) {
-            printf("Unable to run HTTP discovery on ON_FORCE_SYNC \r\n");
+            printf("IOTC: Unable to run HTTP discovery on ON_FORCE_SYNC \r\n");
             return;
         }
         sync_response = run_http_sync(config.cpid, config.duid);
         if (NULL == sync_response) {
-            printf("Unable to run HTTP sync on ON_FORCE_SYNC \r\n");
+            printf("IOTC: Unable to run HTTP sync on ON_FORCE_SYNC \r\n");
             return;
         }
-        printf("Got ON_FORCE_SYNC. Disconnecting.\r\n");
+        printf("IOTC: Got ON_FORCE_SYNC. Disconnecting.\r\n");
         iotconnect_sdk_disconnect(); // client will get notification that we disconnected and will reinit
```

Comment:
does this need a break;

Reply:
I will move printf to the beginning of the case and remove the iotconnect_sdk_disconnect() function at line 238 and add a break at the end.

Reply:
fixed.